_includes/includes/ui/keyboard-details.php

```diff
@@ -457,12 +457,12 @@ protected static function WriteTitle() {
 <?php
       if(!empty(self::$deprecatedBy)) {
         $dep = self::$deprecatedBy;
-        $name = self::$title;
+        $id = self::$id;
         echo "
           <div class='download deprecated-new'>
             <a class='download-link' href='/keyboards/$dep$session_query_q'><span>Download</span></a>
-            <div class='download-title'>Download the latest version of $name</div>
-            <div class='download-description'>Click the Download button to get the latest version of this keyboard.</div>
+            <div class='download-title'>Keyboard '$id' is obsolete.</div>
+            <div class='download-description'>Click the Download button to get the replacement <span style='font-weight:bold'>$dep</span> keyboard instead.</div>
```

Comment:
So, for example... if someone finds the "Khmer (NiDA)" keyboard in search, would there still be a way to download that keyboard, or would attempting so only result in obtaining "Khmer Angkor" instead?

Reply:
This is what they are shown by default:

![image](https://user-images.githubusercontent.com/4498365/85484926-1b788480-b60b-11ea-90b7-27bb96c94d61.png)

If they click the link `View details for old version of this keyboard` at the bottom there, they'll see:

![image](https://user-images.githubusercontent.com/4498365/85484977-33e89f00-b60b-11ea-9525-799bf17bf3e6.png)

(Or similar)

Note that this is not new functionality -- I've just tweaked the wording because I thought it was slightly unclear...